backend/api/publishing.py

```
@@ -28,14 +29,194 @@
+        
+        return scenario_responses
+        
+    except Exception as e:
+        debug_log(f"Error fetching scenarios: {e}")
+        import traceback
+        debug_log(f"Traceback: {traceback.format_exc()}")
+        raise HTTPException(status_code=500, detail=f"Failed to fetch scenarios: {str(e)}")
+
+@router.get("/drafts/", response_model=List[ScenarioPublishingResponse])
+async def get_draft_scenarios(
+    db: Session = Depends(get_db),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)